gsoc/2019/mentors.md

```diff
@@ -51,10 +63,10 @@ layout: plain
 * Yuka Takahashi [yuka@cern.ch](mailto:yuka@cern.ch) CERN
 * Vassil Vassilev [vvasilev@cern.ch](mailto:vvasilev@cern.ch)
 * Joao Calado Vicente [joao.calado.vicente@cern.ch](mailto:joao.calado.vicente@cern.ch) CERN
+* Tobias Wegner [tobias.wegner@cern.ch](mailto:tobias.wegner@cern.ch) CERN
 * Sandro Wenzel [sandro.wenzel@cern.ch](mailto:sandro.wenzel@cern.ch) CERN
 * Koen Wolters [k.s.wolters@gmail.com](mailto:k.s.wolters@gmail.com) CERN
-* Alaettin Serhan Mete [amete@cern.ch](mailto:amete@cern.ch) UCIrvine
-* Davide Costanzo [davide.costanzo@gmail.com](mailto:davide.costanzo@gmail.com) The University of Sheffield
-* James Catmore [james.catmore@cern.ch](mailto:james.catmore@cern.ch) University of Oslo
-* Paul Gessinger [paul.gessinger@cern.ch](mailto:paul.gessinger@cern.ch) CERN
-* Andreas Salzburger [andreas.salzburger@cern.ch](mailto:andreas.salzburger@cern.ch) CERN
+* Frank Wuerthwein [fkw@ucsd.edu](mailto:fkw@ucsd.edu) UCSD
+* Edward Moyse [edward.moyse@cern.ch](mailto:edward.moyse@cern.ch) University of Massachusetts, Amherst)
```

Comment:
@EdwardMoyse if there are requirements that you cannot fulfill being part of these universities, then you would need to modify your affiliation here and put `HSF`. Same for Riccardo.

Reply:
> I was told to rebase against master ([#486 (comment)](https://github.com/HSF/hsf.github.io/pull/486#discussion_r253569950))

Sorry it may lead to confusion, after running `git rebase upstream/master` you need to run `git push -f origin master` to avoid this. 

> Apologies if this was a mistake! I can make a new PR, but then this discussion will get lost.

Don't worry, just close this and create a new PR with your files, even if you remove your branch the discussion should be still accessible.